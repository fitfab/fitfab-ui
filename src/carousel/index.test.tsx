import { cleanup } from '@testing-library/react'
// import { Carousel } from './index'

afterEach(cleanup)

test('Render Carousel', () => {
    console.log('TODO: Render Carousel test')
    expect('todo').toBeTruthy()
})
