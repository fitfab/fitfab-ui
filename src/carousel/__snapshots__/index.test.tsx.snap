// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Carousel 1`] = `
<button
  class="sc-bdVaJa dkmxRm"
>
  hello
</button>
`;
